.github/workflows/ci.yml

```diff
@@ -11,25 +11,68 @@ on:
     branches:
       - "**:**"
 
+env:
+  DOCKER_COMPOSE: docker-compose -f docker-compose.yml -f docker-compose.ci.yml
```

Comment:
Not sure if it'd be cleaner to start by `mv docker-compose.ci.yml docker-compose.override.yml` instead of having to remember using the env variable everywhere?

Reply:
We'll just to not forget to repeat the copy in all jobs (except `build`).

Reply:
Could we just use an *actual* environment variable to configure the compose file? [`COMPOSE_FILE`](https://docs.docker.com/compose/reference/envvars/#compose_file) should do.

Reply:
I just copied the file. That's simple.

Reply:
Oh, the `COMPOSE_FILE` is a nice idea. I'll try that!